bin/run.sh

```diff
@@ -0,0 +1,9 @@
+#/bin/sh
+
+set -o errexit
+set -o pipefail
+
+cd tock
+python manage.py migrate --settings=tock.settings.production --noinput
```

Comment:
We can wrap this line with something like `if [ "${CF_INSTANCE_INDEX"} == "0" ]` to limit migrations to the 0th instance.